README.md

```diff
@@ -2,16 +2,43 @@
 
 Source plugin for pulling documents into Gatsby from a Strapi API.
 
-> ⚠️ This version of `gatsby-source-strapi` is only compatible with Strapi v3 at the moment. We are currently working on a v4 compatible version.
+> ⚠️ This version of `gatsby-source-strapi` is only compatible with Strapi v4.
```

Comment:
Maybe we can link to the [npm release](https://www.npmjs.com/package/gatsby-source-strapi/v/1.0.2) compatible with Strapi v3 for those who need it